app/controllers/groups_controller.rb

```diff
@@ -6,4 +6,17 @@ def show
     group = Group.find(params[:id])
     render json: group, serializer: Groups::ShowSerializer
   end
-end
\ No newline at end of file
+
+  def create
+    group = Group.new(params.require(:group).permit(:name))
```

Comment:
This is not following convention. We've been sanitizing params in a separate method before.

Reply:
Right, ... We've refactored this now.